lib/terrafying/components/vpc.rb

```diff
@@ -186,11 +186,11 @@ def create(name, raw_cidr, options = {})
                                                     to_port: 22,
                                                     protocol: 'tcp',
                                                     cidr_blocks: [@cidr],
-                                                    description: nil, 
-                                                    ipv6_cidr_blocks: nil, 
-                                                    prefix_list_ids: nil, 
+                                                    ipv6_cidr_blocks: nil,
+                                                    prefix_list_ids: nil,
                                                     security_groups: nil,
-                                                    self: nil
+                                                    self: nil,
+                                                    description: nil
```

Comment:
is this just a reorder? seems to be the same fields in both

Reply:
Yup, inadvertent reorder. Will remove this